src/components/organisms/Sections/Features/index.jsx

```diff
@@ -9,82 +9,58 @@ import { MDBContainer, MDBRow, MDBCol } from "mdbreact";
 //> CSS
 import "./features.scss";
 
-//> Apollo
-import { gql } from "apollo-boost";
-import { Query } from "react-apollo";
-
-//> Queries
-// Get image by ID
-const GET_IMAGE = gql`
-	query getImage($token: String!, $id: Int!) {
-		image(token: $token, id: $id) {
-			urlLink
-		}
-	}
-`;
-
 class Features extends React.Component {
-	renderReasons = (data) => {
-		let width =
-			data.whyColumns.length === 4 || data.whyColumns.length === 8 ? "3" : "4";
+  renderReasons = (data) => {
+    let width =
+      data.whyColumns.length === 4 || data.whyColumns.length === 8 ? "3" : "4";
 
-		let rtn = data.whyColumns.map((reason, i) => {
-			return (
-				<MDBCol md={width} key={i}>
-					<Query
-						query={GET_IMAGE}
-						variables={{
-							token: localStorage.getItem("jwt"),
-							id: reason.value.Column_image,
-						}}
-						client={this.props.client}
-					>
-						{({ loading, error, data }) => {
-							if (loading) return null;
-							if (error) return null;
-							return (
-								<img
-									src={process.env.REACT_APP_BASEURL + data.image.urlLink}
-									className="img-fluid"
-									alt={reason.value.Column_head}
-								/>
-							);
-						}}
-					</Query>
-					<h2>{reason.value.Column_head}</h2>
-					<p
-						className="lead"
-						dangerouslySetInnerHTML={{ __html: reason.value.Column_subhead }}
-					></p>
-					<p
-						dangerouslySetInnerHTML={{ __html: reason.value.Column_paragraph }}
-					></p>
-				</MDBCol>
-			);
-		});
+    let rtn = data.whyColumns.map((reason, i) => {
```

Comment:
Please add a newline.

Reply:
Prettier does not approve. Wouldn't make sense though. There is a single return in this map function without anything else going on.